src/actions/index.js

```diff
@@ -113,17 +115,31 @@ export function updateCurrentContact(contact) {
 }
 
 /** CONTACTS */
-export function fetchContacts() {
+export function refreshContacts(contacts) {
     return {
-        type: FETCH_CONTACTS,
+        type: REFRESH_CONTACTS,
```

Comment:
Why `refresh` instead of `fetch`? `fetch` to me implies going to an endpoint for data while `refresh` could be app-internal. I'd recommend using `fetch` and `persist` for endpoint related getting and setting, respectively. 

#recommended

Reply:
this diff is misleading, what actually happened here is that I removed the `fetchContacts` method, and left the `refreshContacts` method alone. It was there to begin with from an unrelated change.

In reality, I think it can just be removed since we're not using it